pkg/scheduler/scheduler.go

```diff
@@ -247,7 +248,35 @@ func New(client clientset.Interface,
 		time.Duration(options.bindTimeoutSeconds)*time.Second,
 	)
 
-	registry := frameworkplugins.NewInTreeRegistry()
+	var extenders []core.SchedulerExtender
+	var ignoredResources []string
+	if len(options.extenders) != 0 {
+		var ignorableExtenders []core.SchedulerExtender
+		var ignoredExtendedResources []string
+		for ii := range options.extenders {
+			klog.V(2).Infof("Creating extender with config %+v", options.extenders[ii])
+			extender, err := core.NewHTTPExtender(&options.extenders[ii])
+			if err != nil {
+				return nil, err
+			}
+			if !extender.IsIgnorable() {
+				extenders = append(extenders, extender)
+			} else {
+				ignorableExtenders = append(ignorableExtenders, extender)
+			}
+			for _, r := range options.extenders[ii].ManagedResources {
+				if r.IgnoredByScheduler {
+					ignoredExtendedResources = append(ignoredExtendedResources, r.Name)
+				}
+			}
+		}
+		ignoredResources = ignoredExtendedResources
+
+		// place ignorable extenders to the tail of extenders
+		extenders = append(extenders, ignorableExtenders...)
+	}
+
+	registry := frameworkplugins.NewInTreeRegistry(frameworkplugins.WithIgnoredResources(ignoredResources...))
```

Comment:
Uhm... maybe we should move this until after we have processed extenders in `createFromConfig`.

Reply:
I think we need the registry before that... `createFromConfig` calls `create`, which calls `buildFramework` (a wrapper for `framework.NewFramework`, which takes the registry as a param)